### PR TITLE
SEO-183463 JavaScript Missing H1

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,
Have worked on Missing H1, please review.
Regards,
Yvone.

### CODE REVIEW

File: js/WebAPI/DocumentEditor.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: webAPI reference for DocumentEditor
-description: webAPI reference for 
+title: WebAPI reference in JavaScript DocumentEditor Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline js/WebAPI/DocumentEditor.md:3]: Update like this - WebAPI Reference in JavaScript Document Editor Control | Syncfusion

File: js/WebAPI/DocumentEditor.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+description: Learn here all about webAPI reference support in Syncfusion JavaScript DocumentEditor control, its elements and more.
 documentation: ug
 platform: js
 keywords: documenteditor,syncfusion,documenteditor webapi
 api: /api/js/ejdocumenteditor
 ---
 
-## Import
+# Import in JavaScript DocumentEditor Control
```
Comment (gowthamanthavasiyappan) [inline js/WebAPI/DocumentEditor.md:11]: Update like this - WebAPI Reference in JavaScript Document Editor Control

File: js/WebAPI/DocumentEditor.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: webAPI reference for DocumentEditor
-description: webAPI reference for 
+title: WebAPI reference in JavaScript DocumentEditor Control | Syncfusion
+description: Learn here all about webAPI reference support in Syncfusion JavaScript DocumentEditor control, its elements and more.
```
Comment (gowthamanthavasiyappan) [inline js/WebAPI/DocumentEditor.md:4]: Update like - Learn here all about WebAPI reference support in Syncfusion JavaScript Document Editor control, its elements, and more.

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: js/WebAPI/DocumentEditor.md (hunk 1 above)
Comment (Yvone-Atieno) [reply js/WebAPI/DocumentEditor.md:3]: resolved

File: js/WebAPI/DocumentEditor.md (hunk 3 above)
Comment (Yvone-Atieno) [reply js/WebAPI/DocumentEditor.md:4]: resolved

File: js/WebAPI/DocumentEditor.md (hunk 2 above)
Comment (gowthamanthavasiyappan) [reply js/WebAPI/DocumentEditor.md:11]: Resolved

Comment (Yvone-Atieno) [reply js/WebAPI/DocumentEditor.md:11]: resolved

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**